### PR TITLE
address some compiler warnings

### PR DESCRIPTION
**Pull request by zingale**
## PR checklist

- [ ] test suite needs to be run on this PR
- [ ] this PR will change answers in the test suite to more than roundoff level
- [ ] all newly-added functions have docstrings as per the coding conventions
- [ ] the `CHANGES` file has been updated, if appropriate
- [ ] if appropriate, this change is described in the docs

### CODE REVIEW

**Comment by zingale** [comment]
tests: http://groot.astro.sunysb.edu/Castro/test-suite/gfortran/2020-07-23-002/index.html

File: Source/hydro/riemann.cpp [hunk 1]
```diff
@@ -81,11 +81,11 @@ Castro::cmpflx_plus_godunov(const Box& bx,
       int is_shock = 0;
 
       if (idir == 0) {
-        is_shock = shk(i-1,j,k) + shk(i,j,k);
+        is_shock = static_cast<Real>(shk(i-1,j,k) + shk(i,j,k));
```
Comment (maxpkatz) [inline Source/hydro/riemann.cpp:84]: Do you mean static_cast<int>?

Comment (zingale) [reply Source/hydro/riemann.cpp:84]: oh yes

Comment (zingale) [reply Source/hydro/riemann.cpp]: updated

**Pull request merged**